rql/query/src/main/java/org/eclipse/ditto/rql/query/LikeHelper.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2021 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.ditto.rql.query;
+
+import java.util.regex.Pattern;
+
+public final class LikeHelper {
```

Comment:
Being in a public model, this class would:
* need a javadoc
* need a `@since 2.3.0` javadoc annotation